<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RedFlag</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Users/Arun/Downloads/LogoMakr_4Ox8H5.png</normaloff>../../Users/Arun/Downloads/LogoMakr_4Ox8H5.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_whole">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_sql">
     <attribute name="title">
      <string>SQL Injection</string>
     </attribute>
     <widget class="QLabel" name="label_url_sql">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Url :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_url_sql">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_ok">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_cancel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_sql">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>381</width>
        <height>221</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Help?&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:10pt; color:#222222; background-color:#ffffff;&quot;&gt;SQL injection&lt;/span&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:10pt; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt; is a code &lt;/span&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:10pt; color:#222222; background-color:#ffffff;&quot;&gt;injection&lt;/span&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:10pt; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt; technique, used to attack data-driven applications, in which malicious &lt;/span&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:10pt; color:#222222; background-color:#ffffff;&quot;&gt;SQL&lt;/span&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:10pt; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt; statements are inserted into an entry field for execution (e.g. to dump the database contents to the attacker).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_ddos">
     <attribute name="title">
      <string>DDOS</string>
     </attribute>
     <widget class="QLineEdit" name="input_url_ddos">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_url_ddos">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Url :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_ok_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_cancel_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_ddos">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>381</width>
        <height>221</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Help?&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:10pt; color:#222222; background-color:#ffffff;&quot;&gt;Slowloris&lt;/span&gt;&lt;span style=&quot; font-family:'arial,sans-serif'; font-size:10pt; font-weight:400; color:#222222; background-color:#ffffff;&quot;&gt; is a type of denial of service attack tool invented by Rathienth Baskaran which allows a single machine to take down another machine's web server with minimal bandwidth and side effects on unrelated services and ports.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_output">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output :</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_screen">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>711</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_graph">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
